Makefile

```diff
@@ -521,8 +517,8 @@ endef
 .PHONY: ci-setup-qemu
 ci-setup-qemu:
 	$(call ci_setup_helper,\
-		cd tools/qemu && [[ $$(git rev-parse --short HEAD) == "1ef6d40" ]] && [ -x riscv32-softmmu ] && echo yes,\
-		Clone QEMU fork (with riscv fixes) and run its build scripts,\
+		cd tools/qemu && [ -x riscv32-softmmu ] && echo yes,\
```

Comment:
Hmm.. this actually wasn't quite right before or now. The goal of this command is to efficiently check whether the setup rule needs to run. It should probably be something more of a combination of `git submodule status` and `make -q riscv32-softmmu`.